lib/fhmongodb.js

```diff
@@ -30,6 +30,17 @@ var Database = function (host, port, pDriverOptions, retryWaitTime, dbname) {
 
 util.inherits(Database, EventEmitter);
 
+/**
+ * Get the MongoDB database instance.
+ * 
+ * Will return `null` if connection has not been established.
+ * 
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the useless trailing whitespaces at the end of this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ATrailingWhitespace)